### PR TITLE
Naive build command

### PR DESCRIPTION
**Pull request by roblesch**
This PR wraps the project in a `package.json` and arranges files to be in static and src directoires. This PR introduces a build command that can be executed with `npm run build`. It concats all the dependencies and moves them to the static assets folder in `src/static`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "new-scrubbing",
+  "version": "1.0.0",
+  "description": "Video preview or scrubbing without backend workflow",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "find ./src -maxdepth 1 -iname '*.js' -not -name 'ampPlayer.js' -exec cat {} +> src/static/js/bundle.js && cp src/ampPlayer.js src/static/js/"
```
Comment (phongcao) [inline package.json:8]: This is really clever! I'm having two questions:

1. Is the `find` command works on both Windows and Linux?
2. Can you add the minifier step?

**Review by phongcao — CHANGES_REQUESTED** [review]

File: src/static/Preview.html [hunk 2]
```diff
@@ -12,14 +12,12 @@
...
     <!--*****END OF Azure Media Player scripts*****-->
+
     <!--*****START OF AMP plugin scripts*****-->
-    <script src="js/amp.preview.js"></script>
     <script src="js/ampPlayer.js"></script>
-    <script src="js/browserutils.js"></script>
-    <script src="js/globals.js"></script>
-    <script src="js/mpdutils.js"></script>
-    <script src="js/mseutils.js"></script>
+    <script src="js/bundle.js"></script>
```
Comment (phongcao) [inline src/static/Preview.html:16]: Is there any reason to keep `ampPlayer.js` file outside of the bundle file?

File: src/static/Preview.html [hunk 3]
```diff
@@ -12,14 +12,12 @@
...
     <!--*****END OF AMP plugin scripts*****-->
+
```
Comment (juzuluag) [inline src/static/Preview.html]: Nit: Remove space

File: package.json (hunk 1 above)
Comment (roblesch) [reply package.json:8]: This build step will only work in bash. I can write a js script to move & minify that should work fine xplat, is it ok to do that in an additional PR to unblock Dan's build pipeline experimentation?

**Review by juzuluag — COMMENTED** [review]
In general looks promising. I'm in agreement with Phong comments.

File: src/static/Preview.html (hunk 2 above)
Comment (roblesch) [reply src/static/Preview.html:16]: I don't know why, but ampPlayer is unable to load the preview plugin if they are in the same file. https://amp.azure.net/libs/amp/latest/docs/index.html#writing-plugins has a specific note about the order in which scripts are loaded stating that plugins must be loaded after the AMP script

File: src/static/Preview.html (hunk 3 above)
Comment (roblesch) [reply src/static/Preview.html]: :+1:

File: src/static/Preview.html (hunk 2 above)
Comment (juzuluag) [reply src/static/Preview.html:16]: Yup, Seems like the component (amp player) is first initialized in the browser, then the plugin is 'extra' and it has to be done after the instantiation.

File: package.json (hunk 1 above)
Comment (phongcao) [reply package.json:8]: Yes, agree. Let's capture this in another task.

File: src/static/Preview.html (hunk 2 above)
Comment (phongcao) [reply src/static/Preview.html:16]: Got you, just curious.

**Review by phongcao — APPROVED** [review]

**Comment by juzuluag** [comment]
LGTM. Thanks.

**Review by juzuluag — APPROVED** [review]

**Pull request merged**